design/oss_suite_release.md

```diff
@@ -0,0 +1,300 @@
+  - [Summon](https://github.com/cyberark/summon)
+    - [Summon AWS Secrets Provider](https://github.com/cyberark/summon-aws-secrets)
+    - [Summon Chef Databags Provider](https://github.com/cyberark/summon-chefapi)
+    - [Summon Conjur Provider](https://github.com/cyberark/summon-conjur)
+    - [Summon Keyring Provider](https://github.com/cyberark/summon-keyring)
+    - [Summon S3 Provider](https://github.com/cyberark/summon-s3)
+
+## Considerations
+
+### How CyberArk Dynamic Access Provider (DAP) Will Use the OSS Release
+At current, when a new DAP appliance is built it is based on a manually pinned
+version of Conjur OSS. We propose implementing automation as part of this release
+to auto-update the pinned version of Conjur OSS used in DAP with each new OSS
+Suite Release, so that DAP stays consistent with the latest stable release of
+Conjur OSS.
+
+### Open Questions
+
+- How do we ensure new documented features available in the Conjur OSS Suite Release
+  are appropriately propagated to the DAP documentation?
+  - We propose this is out of scope for this effort, and should be decided in a
+    follow-on effort by the DAP team together with the technical writers.
+  - Based on the way documentation currently works, it is likely that new features
+    in the Conjur OSS suite will require additional testing and documentation to
+    certify the new features for DAP in follow-on efforts.
+- How will DAP (built on the open Conjur core) run tests against the non-server
+  components of the Conjur OSS suite to ensure the enterprise product remains
+  compatible with the OSS components that are certified for DAP?
+  - We propose this is out of scope for this effort, and should be decided in a
+    follow-on effort by the DAP team (similarly to how the documentation changes
+    will work abovce).
+- What certification level will each component in the Conjur OSS Suite Release have?
+  Should this be evaluated as part of this effort?
+- What should the initial scope of the automated tests for the Conjur OSS suite
+  release be?
+  - Proposal: The OSS release suite should include an end-to-end test for a single
+    use case: Deploy Conjur OSS with Helm Chart to GKE and enable an app to connect
+    to its database via Secretless using credentials pulled from Conjur.
+  - Individual repos are responsible for running their own integration tests against
+    Conjur. The suite release test suite will enable integration testing multiple
+    components together (not just Component X + Conjur), and as part of the initial
+    effort we'll implement one such use case and ensure we're set up to add other
+    such use cases as we identify them in the future.
+  - Should the scope of this project include evaluating the component repositories
+    to determine the state of the automated tests and whether additional integration
+    tests need to be added?
+  - When a new suite release is being prepared, is there a way to run the automated
```

Comment:
I left this open for now - this is worth discussing, and it's TBD how we would implement this

Reply:
I think this is what you're suggesting... but I would envision (as a future goal) a process where the "candidate release" testing for each component is done on a regular basis, and maybe a test of a candidate release of the entire suite is done e.g. weekly, based on latest candidate releases of each component, so that we're not surprised when it comes time for a release.